src/cpuinfo.cc

```diff
@@ -1,7 +1,5 @@
-#include "cpuinfo.h"
-
 #include<iostream>
-
+#include "cpuinfo.h"
```

Comment:
Why have you changed order of includes? Our coding style convention requires that in xxx.cc a corresponding xxx.h is a first file to include (see: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)